lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2119,17 +2138,21 @@ Instruction *SPIRVToLLVM::transEnqueueKernelBI(SPIRVInstruction *BI,
   }
 
   // Create call to enqueue kernel BI
-  SmallVector<Value *, 8> Args = {
+  SmallVector<Value *, 3> Args = {
```

Comment:
Same here, I think this can be left at 8.

Reply:
Good point. Fixed.